examples/balanced_neuron_example.py

```diff
@@ -0,0 +1,67 @@
+from scipy.optimize import bisect
+import matplotlib.pyplot as plt
+import pyNN.nest as sim
+from pyNN.utility.plotting import Figure, Panel
+
+import pyNN as testpynn
+
+spec_dict = {
+    "simulation_time": 25000.0, # (ms)
+    "n_ex": 16000, # Number of excitatory neurons
+    "n_in": 4000, # Number of inhibitory neurons
+    "r_ex": 5.0, # (Hz) Excitatory Rate
+    "r_in": 20.5, # (Hz) Inhibitory Rate
+    "ex_syn": 0.045, # (nA) Excitatory Synaptic Current Amplitude
+    "in_syn": -0.045, # (nA) Inhibitory Synaptic Current Amplitude
+    "delay": 1.0, # (ms)
+    "low": 15.0, # (Hz)
+    "high": 25.0, # (Hz)
+    "precision": 0.01 
+}
+
+def output_rate(spec_dict, guess):
+
+    newsim = testpynn.nest
+    newsim.setup(timestep=0.1)
```

Comment:
I suggest moving the network creation outside the loop, as is done in the [PyNEST implementation](https://nest-simulator.readthedocs.io/en/stable/auto_examples/balancedneuron.html). Inside the loop, you just need to change the input rate, use `sim.run(...)`, then `neuron.get_data(clear=True)` (the `clear=True` argument removes the data from the population after you've retrieved them)

Reply:
For `balanced_neuron_example`, my results still differ from native NEST example? Is it fine? Is it due to the core implementation being reason for two different resultant value?